packages/idyll-components/src/cite.js

```diff
@@ -0,0 +1,112 @@
+const React = require('react');
+
+const citations = [];
+const referenceInstances = [];
+
+const citationsChangeHandler = {
+  set: function(target, property, value) {
+    referenceInstances.forEach(referenceInstance => {
+      referenceInstance.forceUpdate();
+    });
+    target[property] = value;
+    return true;
+  }
+};
+
+var citationsProxy = new Proxy(citations, citationsChangeHandler);
+
+class References extends React.Component {
+  constructor() {
+    super();
+    referenceInstances.push(this);
+  }
+  render() {
+    return (
+      <div id="references">
+        <h1>References</h1>
+        <ol>
+          {citations.map((citation, index) => (
+            <li key={index}>
+              {citation.authors}:{' '}
+              <a href={citation.url} target="_blank">
+                {citation.title}
+              </a>
+              , {citation.date}
+            </li>
```

Comment:
Let's change the format to:

---

[Title with optional link](https://idyll-lang.org), Author Names. Optional Venue. Optional Date. 

---

It should gracefully degrade if properties aren't provided, so e.g. if no date is provided there shouldn't be a double period at the end.

Reply:
Just made a small edit in the comment above 👆

Reply:
After change:
![Screenshot from 2019-08-04 03-59-16](https://user-images.githubusercontent.com/14182492/62417427-4614c400-b66c-11e9-853b-7c7a8b8ed147.png)